addons/l10n_ec/models/account_move.py

```diff
@@ -150,6 +150,8 @@ def _get_l10n_ec_identification_type(self):
                 identification_code = "01"
             elif is_dni:
                 identification_code = "05"  # TODO revert to "02" (test purposes only)
+            elif self.l10n_latam_document_type_id.internal_type == 'purchase_liquidation':  # TODO review (this made the govt/XSD accept purchase liquidation XMLs)
```

Comment:
@pepetreshere @jco-odoo @vbe-odoo Discuss during our next meeting :crossed_fingers: ?

Reply:
@jco-odoo done, I've splitted the method into _get_l10n_ec_ats_identification_type() and _get_l10n_ec_edi_identification_type(). (the last method is on l10n_ec_edi)

Reply:
This is OK but we need to rename the method name removing the "ats" from it
@pepetreshere

Reply:
Done!, we could close this comment. I added a line as [follows](https://github.com/odoo/odoo/pull/92472/commits/5586cb6c337e7be890f90b41331e254f2de692e0), didn't removed it because when reviewing the notes from the meeting with Joss found we agreed not to remove but to make something for backwards compatibility.
 
`_get_l10n_ec_identification_type = _get_l10n_ec_ats_identification_type #For backward compatibility, remove in master`